README.md

```diff
@@ -0,0 +1,19 @@
+**MEI 3 to 5**
+=====================================
+
+This repository contains a collection of scripts designed to update MEI 3 files to MEI 5. The current implementation of this repository only supports upgrading MEI files from the Liber Usualis. Future developments may extend this functionality to support additional MEI files.
+
+
+**Prerequisites**
+--------------------------------
+
+The only required dependency is the lxml library. You can i
```

Comment:
Looks like there is a missing phrase here...

Reply:
Fixed